conda_devenv/devenv.py

```diff
@@ -26,10 +26,16 @@ def preprocess_selectors(contents):
 
 def _min_conda_devenv_version(min_version):
     """Checks that the current conda devenv version is at least the given version"""
-    from distutils.version import LooseVersion
+    try:
```

Comment:
I think we should depend on `packaging` explicitly, and not use this fallback at all.

Reply:
OK. I didn't realize it was a external package. I will add the dependency.